src/LineageTree/lineageTree.py

```diff
@@ -1282,9 +1301,8 @@ def read_from_mamut_xml(self, path):
         self.predecessor = {}
         self.track_name = {}
         for track in AllTracks:
-            t_id, l = int(track.attrib["TRACK_ID"]), float(
```

Comment:
Why did you change that?
Is it something that do not exist anymore?

By doing that you might remove some backward compatibilities, it's not good.

You could change by that:
```python
if "TRACK_DURATION" in track.attrib:
    t_id, l = int(track.attrib["TRACK_ID"]), float(
                track.attrib["TRACK_DURATION"]
            )
else:
    t_id = int(track.attrib["TRACK_ID"])
```